data.json

```
@@ -2208,6 +2208,15 @@
                 "JavaScript"
             ],
             "description": "a react-based framework that provides accessibility, modularity, responsiveness, and theming in a tidy package"
-        }
+        },
+        {
+            "name": "Todo List App",
+            "link": "https://github.com/pixelledbot/todo-list-app",
+            "label": "good-first-issue",
+            "technologies": ["JavaScript"],
+            "description": "A simple todo list app built with HTML, CSS, and JavaScript. Features include adding, completing, deleting tasks, and persistent storage using localStorage."
+        }          
+            
     ]
+          
```

```suggestion
    ]
```

Please clean up these unneeded blank lines